src/shared/service-util.c

```diff
@@ -22,9 +22,11 @@ static int help(const char *program_path, const char *service, const char *descr
                "Options:\n"
                "  -h --help                 Show this help\n"
                "     --version              Show package version\n"
+               "%s"
                "\nSee the %s for details.\n"
                , program_path
                , description
+               , bus_introspect ? " --bus-introspect=PATH      Write XML introspection data\n" : ""
```

Comment:
line break this, so that one can see quickly the the description is aligned as above?